src/Microsoft.Build.Sql.Templates/README.md

```diff
@@ -17,10 +17,10 @@ dotnet new sqlproj -h
 ```
 
 
-Creating a new project "AdventureWorksLT" for Azure SQL Database (`-t` or `--target-platform`):
+Creating a new project "AdventureWorksLT" for Azure SQL Database (`-tp` or `--target-platform`):
```

Comment:
Maybe add an initial section about installing the templates?

## Installing the templates

Only needs to be done once.

```bash
dotnet new --install Microsoft.Build.Sql.Templates::0.1.7-preview
```